### PR TITLE
just ProcessActiveQueues

### PR DESCRIPTION
**Pull request by theR4K**

### CODE REVIEW

File: src/audio/AudioManager.cpp [hunk 1]
```diff
@@ -920,12 +938,15 @@ cAudioManager::ProcessActiveQueues()
...
 	}
+	field_6 %= m_nActiveSamples;
 }
 
 void
 cAudioManager::ClearRequestedQueue()
 {
 	for (int32 i = 0; i < m_nActiveSamples; i++) {
+		if (i >= m_nActiveSamples)
+			break;
```
Comment (ShFil119) [inline src/audio/AudioManager.cpp:949]: do we need this?

Comment (theR4K) [reply src/audio/AudioManager.cpp:949]: it's from assembly!

File: src/audio/AudioManager.cpp [hunk 2]
```diff
@@ -411,11 +411,11 @@ cAudioManager::SetDynamicAcousticModelingStatus(uint8 status)
...
-bool
-cAudioManager::CheckForAnAudioFileOnCD() const
-{
-	return SampleManager.CheckForAnAudioFileOnCD();
-}
+//bool
+//cAudioManager::CheckForAnAudioFileOnCD() const
+//{
+//	return SampleManager.CheckForAnAudioFileOnCD();
+//}
```
Comment (Sergeanur) [inline src/audio/AudioManager.cpp:418]: We need this

File: src/audio/AudioManager.cpp (hunk 1 above)
Comment (Sergeanur) [reply src/audio/AudioManager.cpp:949]: It's pretty much an optimized for, we don't need this

File: src/audio/DMAudio.cpp [hunk 3]
```diff
@@ -171,7 +171,7 @@ cDMAudio::SetDynamicAcousticModelingStatus(uint8 status)
 bool
 cDMAudio::CheckForAnAudioFileOnCD(void)
 {
-	return AudioManager.CheckForAnAudioFileOnCD();
+	return 0; //exists only in android version, and used only in symbols
```
Comment (Sergeanur) [inline src/audio/DMAudio.cpp:174]: It's used on PC

**Review by Sergeanur — APPROVED** [review]

**Review by ShFil119 — APPROVED** [review]

**Pull request merged**